app/decorators/event_decorator.rb

```diff
@@ -6,7 +6,11 @@ def initialize(event)
 
   def date
     date = @event.happened_at
-    date.in_time_zone(Report::TIME_ZONE).strftime(Report::DATE_FORMAT)
+    unless date.nil?
+      date.in_time_zone(Report::TIME_ZONE).strftime(Report::DATE_FORMAT)
+    else
+      "NaN"
```

Comment:
Don't use abbreviations like "NaN" (Not-a-number) people without a tech background don't understand that.